spec/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+require "aws-sdk"
+
+describe ManageIQ::Providers::Amazon::CloudManager::EventCatcher::Stream do
+  before do
+    @ems = FactoryGirl.create(:ems_amazon_with_authentication, :name => "us-west-1", :provider_region => "us-west-1")
+
+    @ems_stream = described_class.new(@ems.authentication_userid.inspect,
+                                      @ems.authentication_password.inspect,
+                                      @ems.provider_region.inspect,
+                                      @ems.guid.inspect)
```

Comment:
I don't imagine it ends up mattering, as we're stubbing the important part... but these `inspect` calls seem improbable.